release/src/milestones.ts

```diff
@@ -167,6 +168,10 @@ async function setMilestone({ github, owner, repo, issueNumber, milestone }: Git
     });
   }
 
+  if (ignoreExistingMilestones) {
+    return;
+  }
```

Comment:
normally, if an issue in a release branch has an exisiting milestone, we have some logic to either change the milestone to an older one, or add a comment that the fix will also be released in another milestone. Here, all of the issues that already have milestones should be ones with backported fixes to v50, and we can safely ignore them here.